navigation2/package.xml

```diff
@@ -6,14 +6,14 @@
   <description>
     ROS2 Navigation Stack
   </description>
-  <maintainer email="oregon.robotics.team@intel.com">Oregon Robotics Team</maintainer>
   <maintainer email="stevenmacenski@gmail.com">Steve Macenski</maintainer>
   <license>Apache License 2.0</license>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
   <!-- nav2_bringup doesn't belong for recursive dependencies -->
   <exec_depend>nav2_amcl</exec_depend>
+  <exec_depend>nav2_bringup</exec_depend>
```

Comment:
This must be removed, see note I left 2 lines above

Reply:
Yeah. Just saw the comment ... after finding the build failure. :-(. Anyhow. Fixed now.